src/operator/nn/mkldnn/mkldnn_lrn-inl.h

```diff
@@ -133,13 +134,23 @@ void MKLDNNLRNFwd::_Init(const LRNParam &param,
 
 void MKLDNNLRNFwd::SetDataHandle(const NDArray &in_data,
                                  const NDArray &out_data) {
```

Comment:
why `in_data` is not used?

Reply:
It's used in line 140:
this->SetDataHandle(in_data, out_data_mem);

Reply:
It's used in line 140:
this->SetDataHandle(in_data, out_data_mem);